OrangeHouse.qml

```diff
@@ -1,14 +1,27 @@
 import Qt3D.Core 2.0
 import Qt3D.Render 2.0
 import Qt3D.Extras 2.0
+import QtQuick 2.7
 
 Entity {
-	components: [
-		SceneLoader {
-			source: "/models/orangehouse.qgltf"
-		},
-		Transform {
-			translation: Qt.vector3d(0, 0, -0.038) // the marker is on the top of the house
-		}
-	]
+    property matrix4x4 t
```

Comment:
It would be more helpful to name the transform with its semantic meaning, for instance worldPose.